src/bacnet/basic/service/h_cov.c

```diff
@@ -357,6 +357,7 @@ static bool cov_list_subscribe(BACNET_ADDRESS *src,
     /* unable to subscribe - resources? */
     /* unable to cancel subscription - other? */
 
+    fprintf(stderr, "[%s %d %s]\r\n", __FILE__, __LINE__, __func__);
```

Comment:
Do we want to report this line ever time? There doesn't seem to be a condition around it, or maybe I am just missing it?

Reply:
Yeah an overlook. Sotred.